yeoman-generator/src/app/index.ts

```diff
@@ -28,31 +30,64 @@ export default class extends Generator {
 
   }
 
+  async initializingCheckAlphaBetaTarget() {
+    const answer = await this.prompt({
+      type: "list",
+      name: "hasuraDdnVersion",
+      message: "Which version of the Hasura DDN are you targeting?",
+      choices: [
+        {
+          name: "Hasura DDN Alpha",
+          value: "alpha",
+        },
+        {
+          name: "Hasura DDN Beta",
+          value: "beta",
+        }
+      ]
+    });
+    this._targetHasuraDdnVersion = answer.hasuraDdnVersion;
+  }
```

Comment:
Can prompt responses be pre-baked with flags? Just want to make sure that if a user knows what they need to do then they can skip the prompts.

Reply:
No worries if not. Just a UX flow thought.

Reply:
Yes, we could set a default here, but which would you choose?